tests/CppUTest/UtestPlatformTest.cpp

```diff
@@ -25,6 +25,8 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <stdlib.h>
```

Comment:
I missed out this one. Can you avoid including stdlib directory and instead use the wrapped header for that.

Reply:
what about lines 67, 101, 102?

Reply:
Done, but I have no idea how to wrap exit() call in such case

Reply:
Note that stdlib.h is directly included in `UtestPlatform.cpp` file. Probably whatever is inside `CPPUTEST_HAVE_FORK` later in this file can assume presence of `stdlib.h`, so it (`exit()`) is not an issue.